chart/values.yaml

```diff
@@ -167,6 +167,12 @@ assets:
   # Directory on the shared storage (audio files and images)
   storageDirectory: "/assets"
 
+cachedAssets:
+  # base URL for the cached assets files. It should be set accordingly to the datasets-server domain, eg https://datasets-server.huggingface.co/cached-assets
+  # baseUrl: "not used for now"
+  # Directory on the cached shared storage (audio files and images)
+  storageDirectory: "/cached-assets"
+
```

Comment:
Even if it's not as important as the other variables, we should have all the env vars in this file:

```
CACHED_ASSETS_CLEAN_CACHE_PROBA = 0.05
CACHED_ASSETS_KEEP_FIRST_ROWS_NUMBER = 100
CACHED_ASSETS_KEEP_MOST_RECENT_ROWS_NUMBER = 200
CACHED_ASSETS_MAX_CLEANED_ROWS_NUMBER = 10_000
```